finders/find_name.py

```diff
@@ -1,7 +1,30 @@
 from typing import List
-
 from finders.stops import Stop
+import re
 
 
 def find_stops_matching_name(stops: List[Stop], name: str, regex: bool = False) -> List[Stop]:
-    pass
+    found = False
+    results = list()
+    if not regex:
+        for element in stops:
+            if element.name == name:
+                print("Your stop is: ", element)
+                found = True
```

Comment:
found is not really useful since you return the value as soon as you find the element with the right name
if you exist the loop, you know you didn't find a value